packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -90,84 +90,101 @@ type TAsyncSelectInputProps = {
   // See https://react-select.com/props#select-props
   /**
    * Aria label (for assistive tech)
+   * [Props from React select was used](https://react-select.com/props)
```

Comment:
Let's add a blank line (same for the other props).

```suggestion
   * Aria label (for assistive tech)
   * <br>
   * [Props from React select was used](https://react-select.com/props)
```